src/main.rs

```diff
@@ -99,64 +129,100 @@ fn main() {
+                .long_help(format!("\
+Strip metadata chunks, where <mode> is one of:
+
+safe    =>  Strip all non-critical chunks, except for the following:
+                {}
+all     =>  Strip all non-critical chunks
+<list>  =>  Strip chunks in the comma-separated list, e.g. 'bKGD,cHRM'
+
+CAUTION: 'all' will convert APNGs to standard PNGs.
+
+Note that 'bKGD', 'sBIT' and 'hIST' will be forcibly stripped if the color type or bit \
+depth is changed, regardless of any options set.",
+                    StripChunks::KEEP_SAFE
+                        .iter()
+                        .map(|c| String::from_utf8_lossy(c))
+                        .collect::<Vec<_>>()
+                        .join(", ")))
                 .long("strip")
                 .value_name("mode")
                 .conflicts_with("strip-safe"),
         )
         .arg(
             Arg::new("keep")
-                .help("Strip all optional metadata except objects in the comma-separated list")
+                .help("Strip all metadata except in the comma-separated list")
                 .long("keep")
                 .value_name("list")
                 .conflicts_with("strip")
                 .conflicts_with("strip-safe"),
         )
         .arg(
             Arg::new("alpha")
-                .help("Perform additional alpha optimizations")
+                .help("Perform additional alpha channel optimization")
+                .long_help("\
+Perform additional optimization on images with an alpha channel, by altering the color \
+values of fully transparent pixels. This is generally recommended for better compression, \
+but take care as this is technically a lossy transformation and may be unsuitable for some \
+applications.")
                 .short('a')
                 .long("alpha")
                 .action(ArgAction::SetTrue),
         )
         .arg(
             Arg::new("interlace")
-                .help("PNG interlace type - Default: 0")
+                .help("Set PNG interlacing type (0, 1, keep)")
+                .long_help("\
+Set the PNG interlacing type, where <type> is one of:
+
+0     =>  Remove interlacing from all images that are processed
+1     =>  Apply Adam7 interlacing on all images that are processed
+keep  =>  Keep the existing interlacing type of each image
+
+Note that interlacing can add 25-50% to the size of an optimized image. Only use it if you \
+believe the benefits outweigh the costs for your use case.")
                 .short('i')
                 .long("interlace")
                 .value_name("type")
-                .value_parser(["0", "1", "keep"]),
+                .default_value("0")
+                .value_parser(["0", "1", "keep"])
+                .hide_possible_values(true),
         )
         .arg(
             Arg::new("scale16")
                 .help("Forcibly reduce 16-bit images to 8-bit")
```

Comment:
Tweak: `"Forcibly reduce 16-bit images to 8-bit"` --> `"Forcibly reduce 16-bit per channel images to 8-bit"`

Reply:
Nice, I've expanded the help here a bit more.